build_daemon/CHANGELOG.md

```diff
@@ -1,31 +1,35 @@
-# 2.1.7
+## 2.1.8
```

Comment:
We keep this matching exactly the pubspec so:

```suggestion
## 2.1.8-dev
```

Reply:
done